src/components/routes/homepage/ImagePicker.js

```diff
@@ -31,7 +31,7 @@ export const ImagePicker = ({ imageBase64, setImageBase64 }) => {
       />
       <FormLabel htmlFor="event-image" className={classes.root}>
         <Button startIcon={<CloudUpload />} fullWidth={true} component="span">
-          {imageBase64 ? 'Change image' : 'Upload image'}
```

Comment:
what was the issue here?

Reply:
so initially when we didnt use any placeholder image (initial state of base64 was an empty string), that works fine. but after adding a placeholder, it will always show “change image” even from the beginning since technically the placeholder is still an image)

we could still use the ternary operator but might have to import placeholder here and look like base64 === placeholder ? ... which one do you prefer??

Reply:
I see. `base64===placeholder` seems off, maybe use a `state` instead, might be an overkill

Reply:
alrightt, will make changes soon